tests/exec.go

```diff
@@ -101,7 +101,7 @@ func Exec(o *option.Option) {
 
 					for name, want := range testCases {
 						output := command.StdoutStr(o, "exec", user, name, testContainerName, "id")
-						gomega.Expect(output).Should(gomega.Equal(want))
+						gomega.Expect(output).Should(gomega.ContainSubstring(want))
```

Comment:
Could we update `testCases` to match the exact strings of the new output and use `gomega.Or` as well as add TODOs like how we deal with image SHA below?